src/lib/tools.ts

```diff
@@ -46,3 +72,107 @@ export async function writeYaml(
 	});
 	return obj;
 }
+
+export function gitURI(uri: string): boolean {
+	const re = new RegExp(
+		'((git|ssh|http|https)|(git@[\\w\\.]+))(:(//)?)([\\w\\.@\\:/\\-]+)',
+	);
+	return re.test(uri);
+}
+
+export function localPathURI(uri: string): boolean {
+	const re = new RegExp('^([a-zA-Z0-9_/\\-.])+$');
+	return re.test(uri);
```

Comment:
Same as previous function, just carry out single line test.

Reply:
Again, changed to a more dense style.